src/process/process.cr

```diff
@@ -112,6 +122,16 @@ class Process
     LibC.times(out tms)
     Tms.new(tms.utime / hertz, tms.stime / hertz, tms.cutime / hertz, tms.cstime / hertz)
   end
+
+  def self.daemonize(stdin : String = "/dev/null", stdout : String = "/dev/null", stderr : String = "/dev/null")
+    exit if fork
+    setsid
+    exit if fork
+    Dir.cd("/")
```

Comment:
I referred to http://www.jstorimer.com/blogs/workingwithcode/7766093-daemon-processes-in-ruby as a guide.

Ruby source for Process.daemon might be a good reference as well, but I didn't look at that. Also, http://stackoverflow.com/questions/3095566/linux-daemonize.

The intent of `cd /`is to guarantee that the daemon will continue to run despite file system changes. `/` is guaranteed to always exist. However, I can see current working dir as a legitimate method parameter, defaulting to either `Dir.current` or `/`.